examples/gateway/pix_payment.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * This example shows you how to process a PIX transaction.
+ * You need to redirect user to the url returned by API to let her/him finish the operation
+ */
+
+include __DIR__ . '/../../vendor/autoload.php';
+
+try {
+    $pixPayment = new \VendoSdk\Gateway\PixPayment();
```

Comment:
Wouldn't it make more sense to just have 1 request class per endpoint?
Then I would make the payment method a Request\Details entity...

Reply:
done